<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui_Main</class>
 <widget class="QMainWindow" name="Gui_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keto Event Logger v2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res_main.qrc">
    <normaloff>:/icons/keto_48</normaloff>:/icons/keto_48</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="teamLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="teamNameLabel">
        <property name="text">
         <string>Team name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="teamCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="finishTimeLabel">
        <property name="text">
         <string>Finish time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="finishTime">
        <property name="currentSection">
         <enum>QDateTimeEdit::HourSection</enum>
        </property>
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="teamSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="taskList">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="findTaskLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="findTaskButton">
        <property name="placeholderText">
         <string>Find task by name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="clearButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="res_main.qrc">
          <normaloff>:/icons/clear_22</normaloff>:/icons/clear_22</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionTeams"/>
   <addaction name="actionTasks"/>
   <addaction name="actionRankings"/>
   <addaction name="actionSettings"/>
   <addaction name="actionAbout"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionTeams">
   <property name="icon">
    <iconset resource="res_main.qrc">
     <normaloff>:/icons/teams_48</normaloff>:/icons/teams_48</iconset>
   </property>
   <property name="text">
    <string>Teams</string>
   </property>
   <property name="toolTip">
    <string>Edit team information</string>
   </property>
  </action>
  <action name="actionTasks">
   <property name="icon">
    <iconset resource="res_main.qrc">
     <normaloff>:/icons/tasks_48</normaloff>:/icons/tasks_48</iconset>
   </property>
   <property name="text">
    <string>Tasks</string>
   </property>
   <property name="toolTip">
    <string>Edit tasks</string>
   </property>
  </action>
  <action name="actionRankings">
   <property name="icon">
    <iconset resource="res_main.qrc">
     <normaloff>:/icons/rankings_48</normaloff>:/icons/rankings_48</iconset>
   </property>
   <property name="text">
    <string>Rankings</string>
   </property>
   <property name="toolTip">
    <string>View team rankings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="res_main.qrc">
     <normaloff>:/icons/about_48</normaloff>:/icons/about_48</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About applet</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res_main.qrc">
     <normaloff>:/icons/exit_48</normaloff>:/icons/exit_48</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit applet</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="res_main.qrc">
     <normaloff>:/icons/settings_48</normaloff>:/icons/settings_48</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Change settings</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res_main.qrc">
     <normaloff>:/icons/save_48</normaloff>:/icons/save_48</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save changes</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res_main.qrc"/>
 </resources>
 <connections/>
</ui>
